Lib/idlelib/autocomplete.py

```diff
@@ -4,29 +4,32 @@
 pop up a list of candidates.
 """
 import __main__
+import itertools
 import os
+import re
 import string
 import sys
 
 # Two types of completions; defined here for autocomplete_w import below.
```

Comment:
*Three* types (or modes?) of completions; ...

Reply:
Yes, thanks!
```suggestion
# Three types of completions; defined here for autocomplete_w import below.
```